src/main/drivers/motor.c

```diff
@@ -343,6 +345,16 @@ void motorEnable(void)
     }
 }
 
+float motorEstimateMaxRpm(void)
+{
+    // Empirical testing found this relationship between estimated max RPM without props attached
+    // (unloaded) and measured max RPM with props attached (loaded), independent from prop size
+    float unloadedMaxRpm = 0.01f * getBatteryVoltage() * motorConfig()->kv;
+    float loadDerating = -5.44e-6f * unloadedMaxRpm + 0.944f;
```

Comment:
What do this numbers mean? Maybe define with a comment.

Reply:
They come from empirical testing (explained in the comment above)